openwebui LXC upgrade bug

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

openwebui

### 📂 What was the exact command used to execute the script?

bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/ct/openwebui.sh)"

### 📝 Provide a clear and concise description of the issue.

The script installs fine without ollama added, however post-install if using the built in "upgrade" command I get the following error stating that the ollama command was not found (as it was not installed) not allowing me to upgrade.

environment: line 31: ollama: command not found

[ERROR] in line 31: exit code 0: while executing command OLLAMA_VERSION=$(ollama -v | awk '{print $NF}')



### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 🔄 Steps to reproduce the issue.

Install Open WebUI without ollama and try the upgrade command

### ❌ Paste the full error output (if available).

environment: line 31: ollama: command not found
[ERROR] in line 31: exit code 0: while executing command OLLAMA_VERSION=$(ollama -v | awk '{print $NF}')

### 🖼️ Additional context (optional).

_No response_

--- a/ct/openwebui.sh
+++ b/ct/openwebui.sh
@@ -28,11 +28,11 @@ function update_script() {
     exit
   fi
 
-  OLLAMA_VERSION=$(ollama -v | awk '{print $NF}')
-  if [ -n "$OLLAMA_VERSION" ]; then
-    RELEASE=$(curl -s https://api.github.com/repos/ollama/ollama/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
+  if [ -x "/usr/bin/ollama" ]; then
+    msg_info "Updating Ollama"
+    OLLAMA_VERSION=$(ollama -v | awk '{print $NF}')
+    RELEASE=$(curl -s https://api.github.com/repos/ollama/ollama/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4)}')
     if [ "$OLLAMA_VERSION" != "$RELEASE" ]; then
-      msg_info "Updating Ollama"
       curl -fsSLO https://ollama.com/download/ollama-linux-amd64.tgz
       tar -C /usr -xzf ollama-linux-amd64.tgz
       rm -rf ollama-linux-amd64.tgz

